docs/api/session.md

```diff
@@ -483,6 +483,10 @@ behavior you can use this API to point the dictionary downloader at your own hos
 dictionaries.  We publish a `hunspell_dictionaries.zip` file with each release which contains the files you need
 to host here.
 
+If the files present in `hunspell_dictionaries.zip` are available at `https://example.com/dictionaries/language-code.bdic`
+then you should call this api with `ses.setSpellCheckerDictionaryDownloadURL('https://example.com/dictionaries/')`.  Please
+note the trailing slash.  The URL to the dictionaries is formed as `${baseURL}${filename}`.
```

Comment:
On this line, could we make the `${baseURL}` variable match the name of the parameter variable on line 479?